### ISSUE TITLE
focus indicator not visible for the composite buttons

### ISSUE THREAD

**Issue by alanrenmsft** [microsoft/azuredatastudio#15596]
![image](https://user-images.githubusercontent.com/13777222/120022015-aba01600-bfa0-11eb-87c8-3e250c07a0f8.png)

**Comment by aasimkhan30** [comment]
https://github.com/microsoft/azuredatastudio/pull/15956 Fixed his issue.

**Comment by alanrenmsft** [comment]
@aasimkhan30 next time you can directly close the issue.
